lib/resque/cluster/config.rb

```diff
@@ -62,10 +70,78 @@ def log_errors
 
       private
 
+      def host_maximums
+        case config_type
+        when :separate
```

Comment:
Well, when there's only one left it no longer feels to me like it belongs in a separate object.